### PR TITLE
Implement stringer interface for the Query type

### PR DESCRIPTION
**Pull request by johnweldon**
To make logging a little nicer in downstream usage.

### CODE REVIEW

**Comment by Zariel** [comment]
Hi, thanks for this. It might be worth making this consistent with the other string methods in frame, something like

`[query statment=%q values=%+#v consistency=%s]`

**Comment by johnweldon** [comment]
makes sense - will do
Thanks

**Comment by Zariel** [comment]
Also if you are not already could you add yourself to the AUTHORS file, thanks.

File: session.go [hunk 1]
```diff
@@ -434,6 +434,11 @@ type Query struct {
 	defaultTimestamp bool
 }
 
+// String implements the stringer interface.
+func (q *Query) String() string {
```
Comment (bfontaine) [inline session.go]: Why did you implement it on a pointer receiver?

Comment (johnweldon) [inline session.go]: Good catch @bfontaine - it was actually a mistake I've made for a few months now.  I mis-interpreted [Effective Go](https://golang.org/doc/effective_go.html#pointers_vs_values) and have been "doing it wrong" until you asked the question :)

I'll fix this.

**Comment by johnweldon** [comment]
Okay @bfontaine and @Zariel - I think this is finally ready to merge :)

**Comment by Zariel** [comment]
Thank you for the contribution!

**Pull request merged**